src/sentry/integrations/slack/webhooks/options_load.py

```diff
@@ -69,15 +74,17 @@ def get_filtered_option_groups(
 
         option_groups = []
         if filtered_teams:
-            team_options = format_actor_options(filtered_teams, True)
-            option_groups.append(
-                {"label": {"type": "plain_text", "text": "Teams"}, "options": team_options}
-            )
+            team_options_group: OptionGroup = {
+                "label": {"type": "plain_text", "text": "Teams"},
+                "options": format_actor_options(filtered_teams, True),
+            }
+            option_groups.append(team_options_group)
         if filtered_members:
-            member_options = format_actor_options(filtered_members, True)
-            option_groups.append(
-                {"label": {"type": "plain_text", "text": "People"}, "options": member_options}
-            )
+            member_options_group: OptionGroup = {
+                "label": {"type": "plain_text", "text": "People"},
+                "options": format_actor_options(filtered_members, True),
+            }
+            option_groups.append(member_options_group)
         return option_groups
```

Comment:
It might be better just to update this to `list[OptionGroup]` since we now have a concrete type. Having `Any` in the return type might mask more type issues in the callers.